pkg/report/report.go

```diff
@@ -635,6 +639,10 @@ func Generate(ctx context.Context, path string, mrs *yarax.ScanResults, c malcon
 
 // upgradeRisk determines whether to upgrade risk based on finding density.
 func upgradeRisk(ctx context.Context, riskScore int, riskCounts map[int]int, size int64) bool {
+	if ctx.Err() != nil {
+		return false
+	}
+
```

Comment:
Same here - this doesn't look like an intensive operation that we need to worry about context cancellations.

Reply:
Removed in [`62a0438` (#876)](https://github.com/chainguard-dev/malcontent/pull/876/commits/62a043878f3ddc344f08d705c83b08e1c9425688).